### DISCUSSION TITLE
422 (Unprocessable Content)

### DISCUSSION THREAD

**Discussion by aoguzhut** [firefly-iii/firefly-iii#7674] [Firefly III]
Hello, I did a clean install, I ran the system, I created a budget then I created a bill. It gives an error when I want to create a new expense, but none of the input fields are red. Only this top of the page: "Error! There was something wrong with your submission. Please check out the errors." When I checked the error from console, I saw: 

create_transaction.js?v=6.0.13:2     POST http://******/api/v1/transactions?_token=Mw3tUVPz7MrlNdx5kdrJTUMcIccPrU8ViUBGwv1x 422 (Unprocessable Content)

create_transaction.js?v=6.0.13:2 L {message: 'Request failed with status code 422', name: 'AxiosError', code: 'ERR_BAD_REQUEST', config: {…}, request: XMLHttpRequest, …}

422 Unprocessable Content

{"message":"Unauthenticated","exception":"AuthenticationException"}

**Comment by JC5** [comment]
What happens when you try it in a private browser window?

**Comment by aoguzhut** [comment]
> What happens when you try it in a private browser window?

same result. and i see the Transaction information screen source account list and destination account list are the same, is it normal?

**Reply by aoguzhut** [reply]
Oh, finally, i got it with this nginx sites-enabled config. thank you.
I think it was a case of mod_rewrite in nginx.
```
server {
        listen 80 default_server;
        listen [::]:80 default_server;
        root /var/www/firefly-iii/public/;
        index index.php index.html index.htm index.nginx-debian.html;
        server_name "$hostname";
        include /etc/nginx/sites-dietpi/*.conf;
        location / {
                try_files $uri $uri/ /index.php?$args;
        }
        location ~ \.php(?:$|/) {
                fastcgi_split_path_info ^(.+\.php)(/.+)$;
                include snippets/fastcgi-php.conf;
                fastcgi_pass unix:/var/run/php/php8.2-fpm.sock;
        }
}
```
